examples/switch/switch-button.html

```diff
@@ -0,0 +1,239 @@
+        <th><kbd>Tab</kbd></th>
+        <td>
+          <ul>
+            <li>Moves keyboard focus to the <code>switch</code>.</li>
+          </ul>
+        </td>
+      </tr>
+      <tr data-test-id="key-space">
+          <th><kbd>Space</kbd>, <kbd>Enter</kbd></th>
+          <td>
+            <ul>
+              <li>Toggle switch between on and off.</li>
+            </ul>
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </section>
+
+  <section>
+    <h2 id="rps_label">Role, Property, State, and Tabindex  Attributes</h2>
+    <table aria-labelledby="rps_label" class="data attributes">
+      <thead>
+        <tr>
+          <th scope="col">Role</th>
+          <th scope="col">Attribute</th>
+          <th scope="col">Element</th>
+          <th scope="col">Usage</th>
+        </tr>
+      </thead>
+      <tbody>
+          <tr data-test-id="h3">
+            <th scope="row"></th>
+            <td></td>
+            <td><code>h3</code></td>
+            <td>
+              <ul>
+                <li>Provides a grouping label for the group of switches.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="group-role">
+            <th scope="row"><code>group</code></th>
+            <td></td>
+            <td><code>div</code></td>
+            <td>
+              <ul>
+                <li>Identifies the <code>div</code> element as a <code>group</code> container for the switches.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="group-aria-labelledby">
+            <th scope="row"></th>
+            <td><code>aria-labelledby</code></td>
+            <td><code>div</code></td>
+            <td>
+              <ul>
+                <li>The <code>aria-labelledby</code> attribute references the <code>id</code> attribute of the <code>h3</code> element to define the accessible name for the group of switches.</li>
+              </ul>
+            </td>
+          </tr>
+          <tr data-test-id="switch-role">
+            <th scope="row"><code>switch</code></th>
+            <td></td>
+            <td><code>button</code></td>
+            <td>
+              The <code>role=&quot;switch&quot;</code> attribute identified the <code>button</code> element as an ARIA <code>switch</code>.
+            </td>
+          </tr>
+          <tr data-test-id="switch-aria-checked">
+            <td></td>
+            <th scope="row"><code>aria-checked=&quot;false&quot;</code></th>
+            <td><code>button</code></td>
+            <td>
+                <ul>
+                  <li>Identifies the <code>switch</code> button as off.</li>
+                  <li>CSS attribute selectors (e.g. <code>[aria-checked=&quot;false&quot;]</code>) are used to synchronize the visual states with the value of the <code>aria-checked</code> attribute.</li>
+                </ul>
+            </td>
+          </tr>
+          <tr data-test-id="switch-aria-checked">
+            <td></td>
```

Comment:
why is this here?

Reply:
@jesdaigle  what are you referring to when you ask about "this"? Asking about the presence of the first cell in the row? The first ceel is for role, which is blank in this row b/c this is a row describing attributes of the row for the switch role.